e2e/rbd.go

```diff
@@ -1466,6 +1466,41 @@ var _ = Describe("RBD", func() {
 				},
 			)
 
+			By(
+				"create a PVC and Bind it to an app with object-map,fast-diff image-features and default mounter",
+				func() {
+					err := deleteResource(rbdExamplePath + "storageclass.yaml")
+					if err != nil {
+						e2elog.Failf("failed to delete storageclass with error %v", err)
```

Comment:
nit: Please drop the `with error` wording, we're trying to remove the redundancy of reporting an error and additionally mentioning `with error`. I understand many existing tests still include it, so it would be nice if you can change it for your tests in an update of this PR.